app/views/complaints/_pagination.html.erb

```diff
@@ -0,0 +1,57 @@
+<%# source: https://designsystem.digital.gov/components/pagination/ %>
+<% page_list = paginate %>
+<% current = params[:page].present? ? params[:page].to_i : 1 %>
+
+<% if page_list.present? %>
+  <nav aria-label="Pagination" class="usa-pagination">
+    <ul class="usa-pagination__list">
+
+      <!-- PREVIOUS -->
+      <% if current > 1 %>
+        <li class="usa-pagination__item usa-pagination__arrow">
+          <%= link_to complaints_path(page: current - 1),
+            class: "usa-pagination__link usa-pagination__previous-page",
+            "aria-label": "Previous page" do
+          %>
+            <svg class="usa-icon" aria-hidden="true" role="img">
+              <use xlink:href="/assets/img/sprite.svg#navigate_before"></use>
+            </svg>
+
+            <span class="usa-pagination__link-text"> Previous </span>
+          <% end %>
+        </li>
+      <% end %>
+
+      <% page_list.each do |item| %>
+        <% if item == "|" %>
```

Comment:
[thought] with a page object, we could just ask `page.first? || page.last?` which is easier to reason about than a `|`

Reply:
➕